chain/network/src/peer_manager.rs

```diff
@@ -150,32 +148,44 @@ impl IbfRoutingTableExchangeActor {
     }
 }
 
-#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
 impl Actor for IbfRoutingTableExchangeActor {
     type Context = SyncContext<Self>;
 }
 
-#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
 #[derive(Debug)]
 pub enum IbfRoutingTableExchangeMessages {
-    AddEdges(Arc<Vec<Edge>>),
+    AddEdges(Vec<Edge>),
     RemoveEdges(Vec<Edge>),
+    #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
     AddPeerIfMissing(PeerId, Option<u64>),
+    #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
     RemovePeer(PeerId),
-    ProcessIbfMessage { peer_id: PeerId, ibf_msg: RoutingVersion2 },
+    #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+    ProcessIbfMessage {
+        peer_id: PeerId,
+        ibf_msg: RoutingVersion2,
+    },
+    RequestRoutingTable,
```

Comment:
I am confused by this change. Shouldn't the entire struct be behind some feature? What is the point of making some of the variants feature gated but not others?

Reply:
`IbfRoutingTableExchangeActor` holds information about `IbfSet` data structures, as well as holds `edges_info` with list of all edges. 
`IbfSet` related data structures are behind ` #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]`.
However, now we will always populate copy of routing table ( `edges_info`) on another thread. That will offload work related to copying routing table for purpose of doing exchange to another thread.

I'll rename `IbfRoutingTableExchangeActor`to `RoutingTableActor` to be more descriptive.

Reply:
Renaming is done